### PR TITLE
Problem: need to add JSON-RPC endpoint personal_initializeWallet

### LINKED ISSUE #738: Problem: need to add JSON-RPC endpoint personal_initializeWallet
personal_initializeWallet is implemented for smartcard wallet initialization.
will return error message currently since smartcard wallet is not supported yet.
refer to: https://github.com/ethereum/go-ethereum/blob/master/internal/ethapi/api.go#L576-L579

### PR DESCRIPTION
**Pull request by Muggle-Du**
Closes #738 

* this is aimed at smartcard wallet which is not supported yet.

## Description

* this is aimed at smartcard wallet which is not supported yet.
* an error with unsupported message will be returned for now.
______

For contributor use:

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: tests/rpc/personal_test.go [hunk 1]
```diff
@@ -148,9 +148,13 @@ func TestPersonal_LockAccount(t *testing.T) {
 func TestPersonal_Unpair(t *testing.T) {
 	t.Skip("skipping TestPersonal_Unpair")
```
Comment (fedekunze) [inline tests/rpc/personal_test.go:149]: I think we shouldn't skip given that we're expecting an error

File: tests/rpc/personal_test.go [hunk 2]
```diff
@@ -148,9 +148,13 @@ func TestPersonal_LockAccount(t *testing.T) {
...
 
-	rpcRes := Call(t, "personal_unpair", []interface{}{"", 0})
+	_, err := CallWithError(t, "personal_unpair", []interface{}{"", 0})
+	require.True(t, errors.Is(err, fmt.Errorf("smartcard wallet not supported yet")))
+}
 
-	var res error
-	err := json.Unmarshal(rpcRes.Result, &res)
+func TestPersonal_InitializeWallet(t *testing.T) {
+	t.Skip("skipping TestPersonal_InitializeWallet")
```
Comment (fedekunze) [inline tests/rpc/personal_test.go:156]: same

**Review by fedekunze — COMMENTED** [review]
LGTM, minor comment. Can you add a changelog entry?

File: tests/rpc/personal_test.go (hunk 2 above)
Comment (Muggle-Du) [reply tests/rpc/personal_test.go:156]: I tried the testcases in personal_test.go with all not skipped for live ethermintd.
every testcase failed for method not registered.
And for testcase like TestPersonal_LockAccount, it expects an error when trying to sign a locked account.
So I just follow the pattern.

Comment (Muggle-Du) [reply tests/rpc/personal_test.go]: add entry in changelog and remove skip

**Review by fedekunze — APPROVED** [review]

**Comment by fedekunze** [comment]
@Muggle-Du can you fix the rpc tests?

```
=== RUN   TestPersonal_Unpair
    personal_test.go:150: 
        	Error Trace:	personal_test.go:150
        	Error:      	Should be true
        	Test:       	TestPersonal_Unpair
--- FAIL: TestPersonal_Unpair (1.00s)
=== RUN   TestPersonal_InitializeWallet
    personal_test.go:155: 
        	Error Trace:	personal_test.go:155
        	Error:      	Should be true
        	Test:       	TestPersonal_InitializeWallet
--- FAIL: TestPersonal_InitializeWallet (1.00s)
```

**Comment by Muggle-Du** [comment]
> @Muggle-Du can you fix the rpc tests?
> 
> ```
> === RUN   TestPersonal_Unpair
>     personal_test.go:150: 
>         	Error Trace:	personal_test.go:150
>         	Error:      	Should be true
>         	Test:       	TestPersonal_Unpair
> --- FAIL: TestPersonal_Unpair (1.00s)
> === RUN   TestPersonal_InitializeWallet
>     personal_test.go:155: 
>         	Error Trace:	personal_test.go:155
>         	Error:      	Should be true
>         	Test:       	TestPersonal_InitializeWallet
> --- FAIL: TestPersonal_InitializeWallet (1.00s)
> ```

ok, I'll check and fix

File: tests/rpc/personal_test.go [hunk 3]
```diff
@@ -146,11 +144,11 @@ func TestPersonal_LockAccount(t *testing.T) {
 }
 
 func TestPersonal_Unpair(t *testing.T) {
-	t.Skip("skipping TestPersonal_Unpair")
-
-	rpcRes := Call(t, "personal_unpair", []interface{}{"", 0})
+	_, err := CallWithError("personal_unpair", []interface{}{"", ""})
+	require.True(t, err.Error() == "smartcard wallet not supported yet")
```
Comment (fedekunze) [inline tests/rpc/personal_test.go:148]:
```suggestion
	require.Equal(t, "smartcard wallet not supported yet", err.Error())
```

File: tests/rpc/personal_test.go [hunk 4]
```diff
@@ -146,11 +144,11 @@ func TestPersonal_LockAccount(t *testing.T) {
...
+}
 
-	var res error
-	err := json.Unmarshal(rpcRes.Result, &res)
-	require.True(t, errors.Is(err, fmt.Errorf("smartcard wallet not supported yet")))
+func TestPersonal_InitializeWallet(t *testing.T) {
+	_, err := CallWithError("personal_initializeWallet", []interface{}{""})
+	require.True(t, err.Error() == "smartcard wallet not supported yet")
```
Comment (fedekunze) [inline tests/rpc/personal_test.go:153]:
```suggestion
	require.Equal(t, "smartcard wallet not supported yet", err.Error())
```

**Pull request merged**